### PR TITLE
[USM] Support postgres alter operation

### PR DESCRIPTION
**Pull request by amitslavin**
### What does this PR do?

Adds support for ALTER operation in Postgres.

### Motivation

Extend our Postgres support capabilities and increase our accuracy.

### CODE REVIEW

**Review by guyarb — COMMENTED** [review]
should make a proper change in pkg/network/encoding/marshal/postgres.go

**Review by guyarb — APPROVED** [review]

**Review by guyarb — APPROVED** [review]

File: pkg/network/usm/postgres_monitor_test.go
Comment (guyarb) [inline pkg/network/usm/postgres_monitor_test.go:1]: what about classification tests?

Comment (amitslavin) [reply pkg/network/usm/postgres_monitor_test.go:1]: already exists - https://github.com/DataDog/datadog-agent/blob/4a49dae97a30986a503a8baabe7fced012c812ea/pkg/network/tracer/tracer_usm_linux_test.go#L1042
The only one missing is truncate

**Pull request merged**